packages/validators/src/utils/common.js

```diff
@@ -14,7 +14,7 @@ export function isObject (o) {
  * @return {*}
  */
 export function unwrap (val) {
-  return isRef(val) ? val.value : val
+  return unref(val)
 }
```

Comment:
Are we keeping this for any particular reason? Should we refactor everything with `unref` now that its available?

Reply:
Yeah, most likely, just haven’t had the time for it yet!